packages/config/lib/options.ts

```diff
@@ -384,7 +384,7 @@ const runtimeOptions: Array<RuntimeConfigOption> = [
     canUpdateDuringTestTime: false,
   }, {
     name: 'clientRoute',
-    defaultValue: '/__/',
+    defaultValue: process.env.CYPRESS_INTERNAL_E2E_TESTING_SELF ? '/__child/' : '/__/',
```

Comment:
One thought was to allow individual test projects to specify those namespaces rather than just hard code them in the openProject. The thought was to provide reasonable defaults so that things would "just work" for developers, but provide the opportunity to override if needed. I am fine with your approach as well though. Might be curious what others thoughts are?

Reply:
@flotwig I think I tried that originally and now I'm realizing why I couldn't do it. I think non-public options can't be passed in on the command line: https://github.com/cypress-io/cypress/blob/10.0-release/packages/server/lib/cypress.js#L33

Reply:
To be cleaner here, I'm going to go with setting things just for the app config and then all of the children will have the default config: https://github.com/cypress-io/cypress/commit/74c25a67e25eebc44b2352d3a85f84c91d2954ff